tools/src/bin/output-file.rs

```diff
@@ -127,7 +129,10 @@ fn read_test_info_from_file_info(per_file_info: Option<&json::Object>) -> Option
                         // I guess this is just covering up our failures?  I'm
                         // sorta tired of this patch though, so... cover up our
                         // failures.
-                        _ => None,
+                        _ => {
+                            warn!("Unhandled disabled condition JSON branch! {:}", cond);
```

Comment:
typo in format string, i think

Reply:
Ah yeah, I'd meant for this to be `{:?}` to get the debug rendering of the condition, unless you meant something else?  I've changed it to `{:?}` locally.

Reply:
Yeah that's what I meant, sorry for unclarity!